pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go

```diff
@@ -121,38 +161,14 @@ func (c *PeriodicBackupController) sync(ctx context.Context, _ factory.SyncConte
 		}
 	}
 
-	if defaultFound {
-		mirrorPods, err := c.podLister.List(labels.Set{"app": "etcd"}.AsSelector())
+	if !defaultFound {
```

Comment:
please add the error handling back and a test for it

Reply:
done 👍🏽